src/common/utils/fhir/formatter.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { ProductDto } from 'src/common/dtos/product.dto';
+
+import * as formatterModule from './formatter';
+
+import { R4 } from '@ahryman40k/ts-fhir-types';
+
+const productDto: ProductDto = {
+  productCode: 'code',
+  productName: 'name',
+  mappings: [
+    { systemName: 'a', systemProductCode: 'code-a', productName: 'prod-a' },
+    { systemName: 'b', systemProductCode: 'code-b', productName: 'prod-b' },
+  ],
+  dateCreated: null,
+  lastUpdated: null,
+};
+
+describe('fhir formatter', () => {
+  describe('getProductsSystems', () => {
```

Comment:
Perhaps you should also add a test to see if it responds correctly to scenarios in which no products have been returned from the master.

Reply:
alright.